### ISSUE TITLE
emb-m-13 - Sidebar

### ISSUE THREAD

**Issue by bertramlyons** [LibraryOfCongress/embARC#30]
emb-m-13	Sidebar		2-High	

"Download" buttons are not visible to screen reader users, and do not receive keyboard focus. They can only be activated using a mouse.	

Ensure these buttons are correctly implemented as interactive elements, receive keyboard focus and are appropriately labeled.

**Comment by bertramlyons** [comment]
Completed and merged in version 1.3.0.
